package/Editor/Plugins/APIPricing.cs

```diff
@@ -5,98 +5,10 @@ namespace Scenario.Editor
-        }
-
-        void DrawInfo()
-        {
-            GUILayout.Label("The Scenario API", EditorStyles.boldLabel);
-            EditorGUILayout.HelpBox("The Scenario API is a versatile tool for image generation and training generators. It's highly adaptable, making it easy to incorporate into games or third-party software. The flexible pricing structure operates on a pay-per-use system, with a monthly billing cycle and cost management options.", MessageType.Info);
-            GUILayout.Label("Pricing Structure", EditorStyles.boldLabel);
-            EditorGUILayout.HelpBox("The API pricing is designed for flexibility and affordability, and begins at a simple rate of 1 cent per image. Costs vary based on image resolution or the use of advanced features. Please refer to the tables below for detailed pricing per image, based on different features and configurations:", MessageType.Info);
-        }
-
-        void DrawTable()
-        {
-            EditorGUILayout.BeginVertical(GUI.skin.box);
-            EditorGUILayout.BeginHorizontal();
-            foreach (string header in headers)
-            {
-                GUILayout.Label(header, EditorStyles.boldLabel);
-            }
-            EditorGUILayout.EndHorizontal();
-            for (int i = 0; i < data.GetLength(0); i++)
-            {
-                EditorGUILayout.BeginHorizontal();
-                for (int j = 0; j < data.GetLength(1); j++)
-                {
-                    GUILayout.Label(data[i, j], GUILayout.Width(70));
-                }
-                EditorGUILayout.EndHorizontal();
-            }
-            EditorGUILayout.EndVertical();
-        }
-
-        void DrawAdditionalCosts()
-        {
-            EditorGUILayout.Space();
-            EditorGUILayout.LabelField("Additional costs (cost per image)", EditorStyles.boldLabel);
-            EditorGUILayout.BeginVertical(GUI.skin.box);
-            for (int i = 0; i < additionalCosts.GetLength(0); i++)
-            {
-                EditorGUILayout.BeginHorizontal();
-                GUILayout.Label(additionalCosts[i, 0], GUILayout.Width(150));
-                GUILayout.Label(additionalCosts[i, 1]);
-                EditorGUILayout.EndHorizontal();
-            }
-            EditorGUILayout.EndVertical();
-        }
-
-        void DrawAdditionalDetails()
-        {
-            EditorGUILayout.Space();
-            EditorGUILayout.LabelField("Additional Details", EditorStyles.boldLabel);
-            EditorGUILayout.BeginVertical(GUI.skin.box);
-            for (int i = 0; i < additionalDetails.GetLength(0); i++)
-            {
-                EditorGUILayout.BeginHorizontal();
-                GUILayout.Label(additionalDetails[i, 0], GUILayout.Width(200));
-                GUILayout.Label(additionalDetails[i, 1]);
-                EditorGUILayout.EndHorizontal();
-            }
-            EditorGUILayout.EndVertical();
+            Application.OpenURL("https://www.scenario.com/api-pricing");
```

Comment:
Better to redirect to: https://docs.scenario.com/page/api-pricing

Reply:
done